docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
It doesn't seem optimal to have a separate makefile for the docs... That and the main project doesn't use make anymore.

Reply:
It's the default generated by sphinx, not too sure if its needed however

Reply:
Let's ditch it.